tests/Browser/Pages/Materials/CreateMaterialTest.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace Tests\Browser\Pages\Materials;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\DB;
+use App\User;
+use App\Category;
+use App\Supplier;
+use App\Material;
+
+class CreateMaterialTest extends DuskTestCase
+{
+    use DatabaseMigrations;
+     
+    /**
+     * Test link Create Material.
+     *
+     * @return void
+     */
+    public function testCreateMaterial()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
+                    ->visit('/materials')
+                    ->click('.fa-plus')
+                    ->assertPathIs('/materials/create')
+                    ->assertSee('Create Material');
+        });
+    }
+
+    /**
+     * Test Validation Create Material.
+     *
+     * @return void
+     */
+    public function testValidationCreateMaterial()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->loginAs(User::find(1))
+                    ->visit('materials/create')
+                    ->press('Create')
+                    ->assertPathIs('/materials/create')
+                    ->assertSee('The name field is required.')
+                    ->assertSee('The category id field is required.')
+                    ->assertSee('The supplier id field is required.')
+                    ->assertSee('The price field is required.')
+                    ->assertSee('The description field is required.');
+        });
+    }
+
```

Comment:
Please add case create a material with a name already has in the database.

Reply:
'The name has already been taken.' cái này có ở phần test testCreateMaterialFailValidation rồi anh.

Reply:
OK em, anh thấy rồi